File: Python PowerPoint Examples/SlidesApiDemo/deleteSlideComment.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.slides_api.SlidesApi(configuration)

fileName = "DeleteSlideComment.pptx"
password = ""
storage = ""
slide_index = 0
comment_index = 0
inputFolder = "input"
result =api.delete_slide_comment(fileName, slide_index=slide_index, comment_index=comment_index, password=password,storage=storage, folder=inputFolder)

File: PHP PowerPoint Examples/SlidesApiDemo/deleteSlideComment.php
<?php

use \Spire\Cloud\Powerpoint\Sdk\Api\SlidesApi;
use \Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new SlidesApi($configuration);

$name = "DeleteSlideComment_1.pptx"; 
$password = null;
$folder = "Slides"; 
$storage = null; 
$slideIndex = 0;
$commentIndex = 0;
$apiInstance->deleteSlideComment($name, $slideIndex,$commentIndex, $password, $folder, $storage);